### PR TITLE
Add choco_install rule

### PR DESCRIPTION
**Pull request by philiparola**
Adds a rule to append '.install' to a chocolatey install command that
failed because of a non-existent package.

### CODE REVIEW

File: tests/rules/test_choco_install.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import pytest
+from thefuck.rules.choco_install import get_new_command
+from thefuck.types import Command
+
+
```
Comment (scorphus) [inline tests/rules/test_choco_install.py]: Tests for `match` are missing. Check other tests for reference.

File: thefuck/rules/choco_install.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+def match(command):
+    return (('choco install' in command.script_parts or 'cinst' in command.script_parts)
+            and 'Installing the following packages' in command.output)
+
+
+def get_new_command(command):
+    cmdList = command.script.split(' ')
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]: `command.script_parts` should be used instead.
```suggestion
```
See below.

File: thefuck/rules/choco_install.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    packageName = ""
+    # Find the argument that is the package name
+    for i in cmdList:
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]: `command.script_parts` should be used instead.

```suggestion
    for script_part in command.script_parts:
```

File: thefuck/rules/choco_install.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+def match(command):
+    return (('choco install' in command.script_parts or 'cinst' in command.script_parts)
+            and 'Installing the following packages' in command.output)
+
+
+def get_new_command(command):
+    cmdList = command.script.split(' ')
+    packageName = ""
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]:
```suggestion
```

File: thefuck/rules/choco_install.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    # Find the argument that is the package name
+    for i in cmdList:
+        print(i)
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]:
```suggestion
```

File: thefuck/rules/choco_install.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        if "choco" in i:
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]: This `if` statement sequence should be rewritten with all expressions grouped together with `and` operators.

File: thefuck/rules/choco_install.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
+def match(command):
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]:
```suggestion
from thefuck.utils import for_app


@for_app("choco", "cinst")
def match(command):
```

File: thefuck/rules/choco_install.py [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    return (('choco install' in command.script_parts or 'cinst' in command.script_parts)
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]:
```suggestion
    return ((command.script.startswith('choco install') or 'cinst' in command.script_parts)
```

Also, the outer parenthesis can be removed.

File: thefuck/rules/choco_install.py [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+        if '=' in i:            # Some paramaters contain '='
+            continue
+        if '/' in i:            # Some parameters contain slashes
+            continue
+        else:
+            packageName = i
+    # Find the name of the broken package, and append metapackage names
+    if not packageName:
+        return False
+    return(command.script.replace(packageName, packageName + ".install"))
```
Comment (scorphus) [inline thefuck/rules/choco_install.py]: Should be disabled when those commands are not available.
```suggestion
    return(command.script.replace(packageName, packageName + ".install"))


enabled_by_default = bool(which("choco")) or bool(which("cinst"))
```

**Comment by scorphus** [comment]
Thanks for your contribution, @philiparola!

**Comment by nvbn** [comment]
Looks good, thanks!

**Pull request merged**